docker/config/config_test.go

```diff
@@ -73,6 +70,8 @@ func TestLoad(t *testing.T) {
 }
 
 func TestLoadWithIrrelevantConfigFile(t *testing.T) {
+	const irrelevantConfigFile = "../../fixtures/docker/config.json.irrelevant"
```

Comment:
could be just `irrelevantConfigFile := "../../fixtures/docker/config.json.irrelevant"` right?

Reply:
yes, `const` here is confusing, right?